project/dimensions/subsectors.csv

```diff
@@ -8,10 +8,10 @@ phev_compact,PHEV Compact Car
 phev_midsize,PHEV Midsize Car
 phev_pickup,PHEV Pickup
 phev_suv,PHEV SUV
-bev_light_medium_truck,BEV Light-Medium Freight Truck
-bev_medium_truck,BEV Medium Freight Truck
-bev_non_freight_truck,BEV Non-Freight Truck
-bev_heavy_truck,BEV Heavy Truck
+bev_light-medium_freight_truck,BEV Light-Medium Freight Truck

```

Comment:
Would you mind always using underscores instead of dashes? This inconsistency is bound to cause user problems.

Reply:
I actually asked for the dash in this case, because it means "light-medium" and not "light and medium".